database.js

```diff
@@ -0,0 +1,31 @@
+const bcrypt = require('bcrypt');
+
+function registerUser(user) {
+  users.push({
+    id: Math.random(),
+    email: user.email,
+    password: bcrypt.hashSync(user.password, 10)
+  });
+}
+
+function getUserByEmail(email) {
+  return users.find(user => user.email === email);
+}
+
+function loginUser(email, password) {
+  const user = getUserByEmail(email);
+  console.log(bcrypt.compareSync(password, user.password))
```

Comment:
please remove console.log from code. its good to use in development but not in production